test/src/test/java/org/corfudb/integration/StateTransferIT.java

```diff
@@ -37,6 +37,7 @@
 import static org.junit.Assert.assertEquals;
 
 @Slf4j
+@SuppressWarnings("checkstyle:magicnumber")
```

Comment:
nit: can be removed

Reply:
These missing annotations don't allow us to compile only the test module. Thats why I feel like keeping them.